src/ui/public/field_editor/__tests__/field_editor.js

```diff
@@ -206,7 +206,7 @@ describe('FieldEditor directive', function () {
         expect(editor.fieldTypes).to.contain('ip');
         expect(editor.fieldTypes).to.not.contain('text');
         expect(editor.fieldTypes).to.not.contain('keyword');
-        expect(editor.fieldTypes).to.not.contain('attachement');
+        expect(editor.fieldTypes).to.not.contain('attachment');
```

Comment:
@spalger Could you take a look at this and advise us on how to proceed? This test fails because `fieldTypes` does contain the string "attachment". Was the original typo deliberate? If not, then did we just uncover a bug with the field editor or is this actually expected behavior? Here's the test failure:

```
08:14:40 Chrome 65.0.3325 (Linux 0.0.0) FieldEditor directive $scope.editor scripted fields provides all kibana types when language is groovy (only possible in 5.x) FAILED
08:14:40 	Error: expected [ 'string',
08:14:40 	  'number',
08:14:40 	  'date',
08:14:40 	  'ip',
08:14:40 	  'boolean',
08:14:40 	  'geo_point',
08:14:40 	  'geo_shape',
08:14:40 	  'attachment',
08:14:40 	  'murmur3',
08:14:40 	  '_source',
08:14:40 	  'unknown',
08:14:40 	  'conflict' ] to not contain 'attachment'
```

Reply:
Since that test is only relevant to 5.x I think it's fine to just make it pass by deleting this line.